requirements-mac.txt

```diff
@@ -2,4 +2,4 @@ wxpython
 pywin32
 pygame
 pyobjc
-pyinstaller==3.6
+pyinstaller==5.13

```

Comment:
Would you mind reverting this specific change?
Probably PyInstaller on Windows works, but I remember the latest pyInstaller did not work on Mac, causing an app bundle error for some unknown reason. Also needs testing for the latest OSX version.